_docs_integrate/data-model-overview.md

```diff
@@ -497,15 +497,16 @@ For more information you should check out the section [AuthenticationRequestItem
+| consent             | `string`                 | The textual consent the user needs to give. This is different to the description of the RequestItem, as the consent would be the actual statement the user agrees to, the description only provides a help text.                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| link                | `string` \| `undefined`  | A valid URL linking to a website which contains more information, like the EULA or privacy terms.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| linkDisplayText     | `string` \| `undefined`  | If a link is specified, a display text, which is displayed in the App instead of the link's URL, can be specified for this link.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| requiresInteraction | `boolean` \| `undefined` | If set to `true`, it advices the recipient of this RequestItem to carefully consider to grant the consent. In particular, this will tell the UI of the App not to preselect the consent checkbox in order to have a user interaction on exactly this consent. This may be necessary for legal reasons, for example, when the consent is required by law or involves the processing of sensitive data.                                                                                                                                                                                                                                |
```

Comment:
> In particular

I don't like this wording here. I'd rather say "In case of the enmeshed App, the UI will bla bla", we don't know how other UIs or non-uis will interpret this.